package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "browserstack-tu-webdriverio",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "on-prem": "npx wdio conf/wdio-on-prem.conf.js",
+    "on-prem-pom": "npx wdio conf/wdio-on-prem-pom.conf.js",
+    "on-prem-datadriven": "npx wdio conf/wdio-on-prem-datadriven.conf.js",
+    "on-prem-parallel": "npx wdio conf/wdio-on-prem-parallel.conf.js",
+    "docker": "npx wdio conf/wdio-docker.conf.js",
+    "docker-parallel": "npx wdio conf/wdio-docker-parallel.conf.js",
+    "browserstack": "npx wdio conf/wdio-browserstack.conf.js",
+    "browserstack-parallel": "npx wdio conf/wdio-browserstack-parallel.conf.js",
+    "browserstack-local": "npx wdio conf/wdio-browserstack-local.conf.js",
```

Comment:
add a command line argument to set any custom session name based on users requirements.
eg: npm run browserstack --test-name="session_name"
This value should then get overridden in the *.conf.js file thats being triggered